spec/http/http_client_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+require 'rack/mock'
+
+describe Twilio::HTTP::Client do
+  before do
+    @client = Twilio::HTTP::Client.new
+  end
+
+  it 'should contain a last response' do
+    expect(Faraday).to receive(:new).and_return(Faraday::Connection.new)
```

Comment:
is the space between `to` and `receive` intentional?

Reply:
Yes.